diff --git a/node_modules/gray-matter/lib/engines.js b/node_modules/gray-matter/lib/engines.js
index 38f993d..bba7b01 100644
--- a/node_modules/gray-matter/lib/engines.js
+++ b/node_modules/gray-matter/lib/engines.js
@@ -13,8 +13,8 @@ const engines = exports = module.exports;
  */
 
 engines.yaml = {
-  parse: yaml.safeLoad.bind(yaml),
-  stringify: yaml.safeDump.bind(yaml)
+  parse: (yaml.load || yaml.safeLoad).bind(yaml),
+  stringify: (yaml.dump || yaml.safeDump).bind(yaml)
 };
 
 /**
